Cargo.toml

```diff
@@ -13,40 +13,34 @@
 # limitations under the License.
 
 [package]
+authors = ["Holger Rapp <hrapp@google.com>"]
```

Comment:
It seems this PR undos a lot of my authors/contributors/version changes....

Reply:
That was accidental, I noticed and fixed it

Reply:
Fixed locally? At least I can't see the fix here...